imagebuildah/build.go

```diff
@@ -389,12 +389,20 @@ func (b *Executor) Run(run imagebuilder.Run, config docker.Config) error {
 // UnrecognizedInstruction is called when we encounter an instruction that the
 // imagebuilder parser didn't understand.
 func (b *Executor) UnrecognizedInstruction(step *imagebuilder.Step) error {
+	err_str := fmt.Sprintf("Build error: Unknown instruction: %q ", step.Command)
+	err := fmt.Sprintf(err_str+"%#v", step)
 	if b.ignoreUnrecognizedInstructions {
-		logrus.Debugf("+(UNIMPLEMENTED KEYWORD %q) %#v", step.Command, step)
+		logrus.Debugf(err)
 		return nil
 	}
-	logrus.Errorf("+(UNIMPLEMENTED KEYWORD %q?) %#v", step.Command, step)
-	return errors.Errorf("Unrecognized instruction %q: %#v", step.Command, step)
+	if logrus.GetLevel() == logrus.ErrorLevel {
+		fmt.Println(err_str)
```

Comment:
I am curious. why are you using `fmt.Println` and not `logrus.Debug` f.e.?

Reply:
Definitely shold not be fmt.Println(err_str).

Either logrus.Debug or logrus.Error.

Reply:
@rhatdan You see it returns a errors.ErrorF at the end? After it falls out of if else.
 Let me see what logrus.Error does.

Reply:
Ok so I tested logrus.ErrorF   (no .Errors) and I will use that instead of the Println. Thanks for the heads up on that.